MapboxNavigation/RouteMapViewController.swift

```diff
@@ -567,6 +568,7 @@ extension RouteMapViewController: NavigationViewDelegate {
         if userTrackingMode == .none, !self.isInOverviewMode {
             self.navigationView.wayNameView.isHidden = true
         }
+        mapView.userTrackingMode = userTrackingMode
```

Comment:
Why did this change?

Reply:
Because otherwise I would need to set the userTracking mode in `startNavigation()`. This way is more reliable. 

If you look at the whole function there seems to be something missing, `userTrackingMode` is read from the mapView, then updated but never written back to the mapView, so it's basically unused. I suspect this was the intent in the first place but could have changed in a previous PR (maybe on accident).

Reply:
I'm not understanding why this would need to change to address the "position camera in bottom 1/3" of the screen logic. Is it required or is this some bonus work?

Reply:
This is done as a precaution to sync the tracking mode when navigating. Why else would we read, then modify the `trackingMode` but not assign it to the mapView? You could even say it fixes something a previous PR missed, otherwise we could remove this function altogether as it does nothing.

![Bildschirmfoto 2024-08-01 um 14 22 26](https://github.com/user-attachments/assets/b095431c-56a5-48e0-b237-90e3fd252f33)

Reply:
I'm not saying you're wrong, but to review your code I need to understand what you are __trying__ to do.

You said this pr is about positioning the puck in the bottom 1/3 of the viewport.

But this change is about something else, right? Did you observe a broken behavior with this? Or is this just about fixing code that "looked wrong"?

Reply:
If you are bothered by this line and want me to remove it because you think it's unnecessary then please say that. I've explained to you twice why I think we need this.